pytest_logger/plugin.py

```diff
@@ -14,6 +15,16 @@
     string_type = str
 
 
+@pytest.hookimpl
```

Comment:
What does the decorator do? Lecture of similar plugins (e.g. catchlog) would suggest that it's not necessary if it doesn't contain arguments.